pypeit/coadd3d.py

```diff
@@ -1101,47 +1144,67 @@ def load(self):
             humidity = self.spec.get_meta_value([spec2DObj.head0], 'humidity')  # Expressed as a percentage (not a fraction!)
             darcorr = DARcorrection(airmass, parangle, pressure, temperature, humidity, cosdec)
 
-            # Compute the extinction correction
-            msgs.info("Applying extinction correction")
-            extinct = flux_calib.load_extinction_data(self.spec.telescope['longitude'],
-                                                      self.spec.telescope['latitude'],
-                                                      self.senspar['UVIS']['extinct_file'])
-            # extinction_correction requires the wavelength is sorted
-            extcorr_sort = flux_calib.extinction_correction(wave_sort * units.AA, airmass, extinct)
+            # TODO :: Need to make a note somewhere that the extinction correction cannot currently be done
+            #         in the datacube because the sensitivity function algorithms correct the standard star
+            #         for extinction when generating the sensitivity function. Including the extinction correction
+            #         in the datacube would result in a double correction of the standard star for extinction.
+            #         This could be wrong when combining multiple standard star exposures if the airmass of the
+            #         standard star exposures is significantly different. For now, to stay consistent with the
+            #         current pipeline, the extinction correction is done in the sensitivity function algorithms,
+            #         with the caveat that the standard star exposures are assumed to have similar airmasses.
+            #         For now, comment out the extinction correction, and reapply this later when the sensitivity
+            #         function algorithms are unified.
```

Comment:
Are there specific things that a user needs to be aware of, or is effectively everything handled correctly behind the scenes?  I.e., adding an issue is definitely useful, but I'm also worried about users not understanding some aspect of the output.

Reply:
Ahh... no, sorry, there is nothing here that the user needs to know. It's primarily a developer note only. However, it might be useful for users that are trying to combine lots of standard star observations at different airmass, so I'll add a temporary note to the documentation.